dcs/action.py

```diff
@@ -1787,6 +1787,166 @@ def dict(self):
+                       horz_alignment, vert_alignment, size, size_units]
+
+    @classmethod
+    def create_from_dict(cls, d, mission):
+        return cls(ResourceKey(d["file"]), d["seconds"], d["clearview"], d["start_delay"],
+                   d["horzAlignment"], d["vertAlignment"], d["size"], d["size_units"])
+
+    def dict(self):
+        return super().dict()
+
+
+class PictureToCoalition(PictureAction):
+    predicate = "a_out_picture_s"
+
+    def __init__(self, coalition: str, file_res_key: ResourceKey, seconds: int,
+                 clearview: bool, start_delay: int,
+                 horz_alignment: str, vert_alignment: str,
+                 size: int, size_units: str):
+        super().__init__(predicate=self.predicate, file_res_key=file_res_key,
+                         seconds=seconds, clearview=clearview,
+                         start_delay=start_delay, horz_alignment=horz_alignment,
+                         vert_alignment=horz_alignment, size=size, size_units=size_units)
+        self.coalition = coalition
+        self.params = [self.coalition, self.file_res_key, seconds, clearview,
+                       start_delay, horz_alignment, vert_alignment, size, size_units]
+
+    @classmethod
+    def create_from_dict(cls, d, mission):
+        return cls(d["coalitionlist"], ResourceKey(d["file"]), d["seconds"], d["clearview"], d["start_delay"],
+                   d["horzAlignment"], d["vertAlignment"], d["size"], d["size_units"])
+
+    def dict(self):
+        d = super().dict()
+        d["coalitionlist"] = self.coalition
+        return d
+
+
+class PictureToCountry(PictureAction):
+    predicate = "a_out_picture_c"
+
+    def __init__(self, country: int, file_res_key: ResourceKey, seconds: int,
+                 clearview: bool, start_delay: int,
+                 horz_alignment: str, vert_alignment: str,
+                 size: int, size_units: str):
+        super().__init__(predicate=self.predicate, file_res_key=file_res_key,
+                         seconds=seconds, clearview=clearview,
+                         start_delay=start_delay, horz_alignment=horz_alignment,
+                         vert_alignment=horz_alignment, size=size, size_units=size_units)
+        self.country = country
+        self.params = [self.country, self.file_res_key, seconds, clearview,
+                       start_delay, horz_alignment, vert_alignment, size, size_units]
+
+    @classmethod
+    def create_from_dict(cls, d, mission):
+        return cls(d["countrylist"], ResourceKey(d["file"]), d["seconds"], d["clearview"], d["start_delay"],
+                   d["horzAlignment"], d["vertAlignment"], d["size"], d["size_units"])
+
+    def dict(self):
+        d = super().dict()
+        d["countrylist"] = self.country
+        return d
+
+
+class PictureToGroup(PictureAction):
+    predicate = "a_out_picture_g"
+
+    def __init__(self, group: int, file_res_key: ResourceKey, seconds: int,
```

Comment:
Same here as for country. Can `Group` be used instead of `int`? If not, `group_id: int`.

Reply:
Same here, changed to group_id.